tests/testsuite/profile_custom.rs

```diff
@@ -21,7 +21,7 @@ fn inherits_on_release() {
         .file("src/lib.rs", "")
         .build();
 
-    p.cargo("build")
+    p.cargo("check")
```

Comment:
I'd also suggest not modifying `profile_custom` as any profile-oriented things might involve codgen.